docs/get-started/create-app.md

```diff
@@ -131,7 +269,7 @@ tensor([[[0.9594, 0.9373],
 
 ## Deploy to JCloud
 
-JCloud offers free CPU and GPU instances to host Jina project. Let's deploy your first project to JCloud:
+JCloud offers free CPU and GPU instances to host Jina project. Let's deploy your first project to JCloud (Note that JCloud is only available for Flow projects):
```

Comment:
As mentioned before, I think we should use [admonition](https://myst-parser.readthedocs.io/en/v0.16.1/syntax/syntax.html#directives-a-block-level-extension-point) instead

Reply:
@alexcg1 ,  Thanks a lot! don't know how I missed that haha